internal/pkg/cli/app_init.go

```diff
@@ -232,6 +235,15 @@ func (o *initAppOpts) validateAppName(name string) error {
 	if err != nil {
 		var noSuchAppErr *config.ErrNoSuchApplication
 		if errors.As(err, &noSuchAppErr) {
+			roleName := fmt.Sprintf("%s-adminrole", name)
+			tags, err := o.iamRoleManager.ListRoleTags(roleName)
+			// return nil when both adminrole and application does not exists then it is a valid app name
```

Comment:
Is "RoleNotFound" the only error that could be returned? What do you think of the following comment?
```go
// NOTE: This is a best-effort attempt to check if the app exists in other regions.
// The error either indicates that the role does not exist, or not. 
// In the first case, it means that this is a valid app name, hence we don't error out.
// In the second case, since this is a best-effort, we don't need to surface the error either.
```